### PR TITLE
(feat) updated simple vwap example

### PR DESCRIPTION
**Pull request by fengtality**
**Before submitting this PR, please make sure**:

- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

**Tests performed by the developer**:

**Tips for QA testing**:

### CODE REVIEW

File: hummingbot/client/config/client_config_map.py [hunk 1]
```diff
@@ -305,22 +305,21 @@ class PaperTradeConfigMap(BaseClientModel):
         default=[
             BinanceConfigMap.Config.title,
             KuCoinConfigMap.Config.title,
-            AscendExConfigMap.Config.title,
+            BitrueConfigMap.Config.title,
```
Comment (cardosofede) [inline hummingbot/client/config/client_config_map.py:308]: are you sure that you want to remove ascendex?

**Review by cardosofede — COMMENTED** [review]
just that observation in the client config map

**Comment by fengtality** [comment]
yes these are the default paper trading exchanges, and I think ascendex
isn't big enough to be a default

**Comment by rapcmia** [comment]
When starting the client from fresh installation, we are getting the error below

image
```
2024-04-19 11:26:29,389 - 7909 - hummingbot.core.utils.trading_pair_fetcher - ERROR - An error occurred when fetching trading pairs for binance_paper_trade.Please check the logs
Traceback (most recent call last):
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/core/utils/trading_pair_fetcher.py", line 51, in fetch_all
    self._fetch_pairs_from_connector_setting(
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/core/utils/trading_pair_fetcher.py", line 40, in _fetch_pairs_from_connector_setting
    connector = connector_setting.non_trading_connector_instance_with_default_configuration()
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/settings.py", line 331, in non_trading_connector_instance_with_default_configuration
    client_config_map=HummingbotApplication.main_application().client_config_map,
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/hummingbot_application.py", line 68, in main_application
    cls._main_app = HummingbotApplication(client_config_map)
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/hummingbot_application.py", line 121, in __init__
    completer=load_completer(self),
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/ui/completer.py", line 505, in load_completer
    return HummingbotCompleter(hummingbot_application)
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/ui/completer.py", line 45, in __init__
    self._spot_exchange_completer = WordCompleter(sorted(AllConnectorSettings.get_exchange_names()), ignore_case=True)
TypeError: '<' not supported between instances of 'NoneType' and 'str'
2024-04-19 11:26:29,431 - 7909 - hummingbot.core.utils.trading_pair_fetcher - ERROR - An error occurred when fetching trading pairs for kucoin_paper_trade.Please check the logs
Traceback (most recent call last):
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/core/utils/trading_pair_fetcher.py", line 51, in fetch_all
    self._fetch_pairs_from_connector_setting(
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/core/utils/trading_pair_fetcher.py", line 40, in _fetch_pairs_from_connector_setting
    connector = connector_setting.non_trading_connector_instance_with_default_configuration()
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/settings.py", line 331, in non_trading_connector_instance_with_default_configuration
    client_config_map=HummingbotApplication.main_application().client_config_map,
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/hummingbot_application.py", line 68, in main_application
    cls._main_app = HummingbotApplication(client_config_map)
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/hummingbot_application.py", line 121, in __init__
    completer=load_completer(self),
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/ui/completer.py", line 505, in load_completer
    return HummingbotCompleter(hummingbot_application)
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/ui/completer.py", line 45, in __init__
    self._spot_exchange_completer = WordCompleter(sorted(AllConnectorSettings.get_exchange_names()), ignore_case=True)
TypeError: '<' not supported between instances of 'NoneType' and 'str'
2024-04-19 11:26:29,470 - 7909 - hummingbot.core.utils.trading_pair_fetcher - ERROR - An error occurred when fetching trading pairs for kraken_paper_trade.Please check the logs
Traceback (most recent call last):
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/core/utils/trading_pair_fetcher.py", line 51, in fetch_all
    self._fetch_pairs_from_connector_setting(
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/core/utils/trading_pair_fetcher.py", line 40, in _fetch_pairs_from_connector_setting
    connector = connector_setting.non_trading_connector_instance_with_default_configuration()
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/settings.py", line 331, in non_trading_connector_instance_with_default_configuration
    client_config_map=HummingbotApplication.main_application().client_config_map,
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/hummingbot_application.py", line 68, in main_application
    cls._main_app = HummingbotApplication(client_config_map)
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/hummingbot_application.py", line 121, in __init__
    completer=load_completer(self),
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/ui/completer.py", line 505, in load_completer
    return HummingbotCompleter(hummingbot_application)
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/ui/completer.py", line 45, in __init__
    self._spot_exchange_completer = WordCompleter(sorted(AllConnectorSettings.get_exchange_names()), ignore_case=True)
TypeError: '<' not supported between instances of 'NoneType' and 'str'
2024-04-19 11:26:29,509 - 7909 - hummingbot.core.utils.trading_pair_fetcher - ERROR - An error occurred when fetching trading pairs for gate_io_paper_trade.Please check the logs
Traceback (most recent call last):
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/core/utils/trading_pair_fetcher.py", line 51, in fetch_all
    self._fetch_pairs_from_connector_setting(
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/core/utils/trading_pair_fetcher.py", line 40, in _fetch_pairs_from_connector_setting
    connector = connector_setting.non_trading_connector_instance_with_default_configuration()
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/settings.py", line 331, in non_trading_connector_instance_with_default_configuration
    client_config_map=HummingbotApplication.main_application().client_config_map,
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/hummingbot_application.py", line 68, in main_application
    cls._main_app = HummingbotApplication(client_config_map)
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/hummingbot_application.py", line 121, in __init__
    completer=load_completer(self),
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/ui/completer.py", line 505, in load_completer
    return HummingbotCompleter(hummingbot_application)
  File "/Users/rapcomia/github/hummingbot/6949/hummingbot/client/ui/completer.py", line 45, in __init__
    self._spot_exchange_completer = WordCompleter(sorted(AllConnectorSettings.get_exchange_names()), ignore_case=True)
TypeError: '<' not supported between instances of 'NoneType' and 'str'
```
- Both `./bin/hummingbot.py` and `./start` return same error 
- Reinstall hummingbot client 2/2 still return same error 

Steps to reproduce:
1. Start the hummingbot client 
2. After you enter the password, the hummingbot client crashes with the reported error above

[logs_hummingbot.log](https://github.com/hummingbot/hummingbot/files/15032779/logs_hummingbot.log)

**Comment by fengtality** [comment]
@rapcmia I think removing OKX from default paper connectors solves this issue

**Comment by rapcmia** [comment]
Test latest commit `88c095a`
- Order created base on the set `filled_order_delay`
- Orders created until `total_amount_quote` is filled then script stops automatically
- User must set if buy or sell on `is_true` param
- Ran tests on mainnet using binance
  - Got same result on paper trade tests
- Ran tests where using a value on `total_amount_quote` exceeds wallet asset. 
   image
   - After we filled all assets, the script still continues with negative timer and at the same time the script returns insufficient balance 

Steps to reproduce:
- Setup script config for simple_vwap.py
- Setup `total_amount_quote` higher than your exchange wallet can place order 
- Start and observe that when it reached `total_amount_quote`, timer gets negative value and script does not stop

PR is good to be approved if the issue on `total_amount_quote` is expected

**Comment by fengtality** [comment]
Thanks @rapcmia I think I'll try to handle the edge cases that you found in another commit

**Comment by fengtality** [comment]
@rapcmia I revised the new VWAP example so that it simply adds configs and removes use of the rate oracle, which seems to cause issues for new users.

**Comment by fengtality** [comment]
I refer to this example in the Botcamp homework assignment, so I will merge this PR to development.

**Pull request merged**